core/src/test/java/org/apache/calcite/test/RelOptRulesTest.java

```diff
@@ -224,6 +224,62 @@ private static boolean skipItem(RexNode expr) {
           && "item".equalsIgnoreCase(((RexCall) expr).getOperator().getName());
   }
 
+  /**
+   * Test case for <a href="https://issues.apache.org/jira/browse/CALCITE-5813">[CALCITE-5813]
+   * Type inference for REPEAT sql function is incorrect</a>. */
```

Comment:
Type inference for sql functions REPEAT, SPACE, XML_TRANSFORM, and XML_EXTRACT is incorrect